Description: Disable sound feature
 because resources have too many copyright holders thus excluded, see
 debcargo.toml.
 Then they are currently baked into bacon, i.e. customized sounds are not
 possible yet, so there's no use leaving it there.
Bug: https://github.com/Canop/bacon/issues/337
Author: Blair Noctis <ncts@debian.org>
Last-Update: 2025-03-06
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -43 +43 @@
-sound = ["rodio"]
+#sound = ["rodio"]
@@ -112 +112 @@
-[dependencies.rodio]
+[disabled.dependencies.rodio]
